styles.css

```diff
@@ -1 +1,80 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+/** {*/
+/*    margin: 0px;*/
+/*    padding: 0px;*/
+/*    box-sizing: border-box;*/
+/*    !*border: 1px solid red;*!*/
+/*}*/
+
+body {
+    background-color: lightgoldenrodyellow;
+    color: floralwhite;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 50px 174px 100px auto 50px;
```

Comment:
Jij zat hier inderdaad beter in de buurt dan ik (240 bij de 2e row). Volgens de uitwerking moet het 190px zijn